dockers/docker-bgp/Dockerfile

```diff
@@ -3,11 +3,15 @@ FROM docker-base
 COPY deps/quagga_*.deb /deps/
 RUN dpkg_apt() { [ -f $1 ] && { dpkg -i $1 || apt-get -y install -f; } || return 1; } &&        \
     dpkg_apt /deps/quagga_*.deb &&                                                              \
+    apt-get update &&                                                                           \
+    apt-get -y install -f python-jinja2 python-netaddr python-ipaddr python-lxml &&             \
     apt-get clean -y && apt-get autoclean -y && apt-get autoremove -y &&                        \
     rm -rf /deps
 
 COPY daemons /etc/quagga/
+COPY ["*.j2", "config.sh", "template_list", "*.py", "/etc/swss/"]
 
```

Comment:
can we embed the mountpoint here, e.g.,

VOLUME ['/etc/sonic']

Then, in the base image, we can put minigraph in the /etc/sonic/,

also, the VOLUME is in baseimage as it is shared by all dockers.